annotate.el

```diff
@@ -58,7 +58,7 @@
 ;;;###autoload
 (defgroup annotate nil
   "Annotate files without changing them."
-  :version "1.6.0"
+  :version "1.6.1"
```

Comment:
This is technically a regression, i.e. a major version change. But that feels silly. It should probably be a minor version change, though, as it is more than a bugfix, right?